docs/src/tutorials/implementing.md

```diff
@@ -241,9 +241,14 @@ Base.unsafe_convert(::Type{Ptr{Cvoid}}, model::Optimizer) = model.ptr
 
 ### Implement methods for `Optimizer`
 
-Now that we have an `Optimizer`, we need to implement a few basic methods.
+All `Optimizer`s must implement the following methods:
 
-* [`empty!`](@ref) and [`is_empty`](@ref)
+ * [`empty!`](@ref)
+ * [`is_empty`](@ref)
+ * [`optimize!`](@ref)
```

Comment:
Or `copy_to_and_optimize!`

Reply:
I'm still not overly sold on this method.